src/common/legacy_config_opts.h

```diff
@@ -413,6 +413,7 @@ OPTION(mds_freeze_tree_timeout, OPT_FLOAT)    // detecting freeze tree deadlock
 OPTION(mds_health_summarize_threshold, OPT_INT) // collapse N-client health metrics to a single 'many'
 OPTION(mds_reconnect_timeout, OPT_FLOAT)  // seconds to wait for clients during mds restart
 	      //  make it (mdsmap.session_timeout - mds_beacon_grace)
+OPTION(mds_cap_revoke_eviction_timeout, OPT_DOUBLE)  // evict client session if it cann't respone revoke msg
```

Comment:
We do not change this file anymore. Please remove this change.

Reply:
ok